modules/javafx.graphics/src/main/java/com/sun/javafx/font/directwrite/DWFactory.java

```diff
@@ -120,7 +121,24 @@ static synchronized IWICImagingFactory getWICFactory() {
         checkThread();
         /* Using single threaded WIC Factory as it should only be used by the rendering thread */
         if (WIC_FACTORY == null) {
+            /* Initialize COM in order to create a WICImagingFactory.
+             * It runs on the prism thread and expects no other codes in this thread
```

Comment:
Typo: codes => code

Reply:
@brcolow 
> Typo: codes => code

Thanks for your review! I've made the change.